highfidelity/hifi
# Blocks app accommodates updated Blocks website

## Test Plan

1. Load blocksApp.js from here: https://hifi-content.s3.amazonaws.com/elisalj/blocks/blocksApp.js
2. Open Blocks app
3. Blocks tablet button should turn white and icon should turn black
4. Choose one of the featured models listed by clicking on it
5. Click the "Download" button: model should appear in front of you
6. Click the upper left menu icon
7. Click "Sign in", then sign in (User: hifi.blocks, Password: downloadz4dayz)
8. Click on the menu icon again, and click "Your Published Items"
9. Click on "purple block"
10. Click on "Download"; a purple block should appear in front of you
11. Click on the Blocks tablet button; tablet should disappear and button should return to black background/white icon
